suite-native/module-send/src/screens/SendReviewScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect, useState } from 'react';
+
+import { CommonActions } from '@react-navigation/native';
+
+import {
+    AppTabsRoutes,
+    GoBackIcon,
+    RootStackRoutes,
+    Screen,
+    ScreenSubHeader,
+    SendStackParamList,
+    SendStackRoutes,
+    StackProps,
+} from '@suite-native/navigation';
+import { Button, Text, VStack } from '@suite-native/atoms';
+import { AccountsRootState, selectAccountByKey } from '@suite-common/wallet-core';
+import { SignedTransaction } from '@trezor/connect';
+import { useToast } from '@suite-native/toasts';
+
+import {
+    onDeviceTransactionReviewThunk,
+    sendTransactionAndCleanupSendFormThunk,
+} from '../sendFormThunks';
+import { ReviewOutputItemList } from '../components/ReviewOutputItemList';
+
+export const SendReviewScreen = ({
+    route,
+    navigation,
```

Comment:
nit: Prefer using `useRoute` and `useNavigation`

Reply:
I believe it's "cleaner", but there is no difference. I'd just like to keep it consistent since we use the hooks almost everywhere